client/src/pages/component/homeBody/Homebody.js

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 import "./Homebody.css";
+
 import { Link } from "react-router-dom";
 
 function Homebody() {
 	return (
 		<>
+			<div className="homebody">
```

Comment:
This div introduced here is not terminated (i.e. it does not have a corresponding `</div>`) and as a result the JSX here does not parse and would break. It might be that these elements have been replaced by other work - if this is the case, you'll need to remove them and check that the web application builds as it does not just now.

Reply:
Hi shieldo, this is not my work this is for Abed, my PR about the navbar only , my file is navbar.js i do  not have this div

Reply:
If you don't have this `div` then you'd need to push your local changes to this branch, or create a new PR. This PR cannot be merged right now because it would break the project.